adapters/Pangle/PangleAdapter/Bidding/GADPangleRTBBannerRenderer.h

```diff
@@ -0,0 +1,24 @@
+//
+//  GADPangleRTBBannerRenderer.h
+//  PangleAdapter
+//
+//  Created by bytedance on 2022/1/11.
+//
+
+#import <Foundation/Foundation.h>
+
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
This can be removed.

Reply:
remove "NS_ASSUME_NONNULL_BEGIN" ?

Reply:
That's correct.

Reply:
Do I need remove all files or the current file ?